devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -114,7 +140,13 @@ def installed() {
 def updated() {
 	// Device-Watch simply pings if no device events received for 32min(checkInterval)
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
-	response(refresh())
+
+	def additionalCmds = []
```

Comment:
Since additionalCmds is filled in case of AeotecNanoDimmer only, is it worth to pass an empty array to response() method in case of other devices ?

Reply:
sure, fixed